src/style.css

```diff
@@ -1 +1,33 @@
-/* add styles here */
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+}
+
+.stars {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  padding-right: 4px;
```

Comment:
do you really need this padding?
```suggestion
```

Reply:
> do you really need this padding?

no, it's just a loss)

Reply:
> do you really need this padding?

![image](https://github.com/mate-academy/layout_stars/assets/144326045/ad6dc914-a244-44d5-b13c-a85b6daabbae)

something went wrong (
after simply removing one line with padding and committing, the task stopped passing the tests
and npm start says that the page was not found
what can it be?